packages/node_modules/@ciscospark/plugin-phone/src/phone.js

```diff
@@ -239,20 +248,26 @@ const Phone = SparkPlugin.extend({
    * @returns {undefined}
    */
   onLocusEvent(event) {
-    if (shouldRing(event) && !this.emittedCalls.has(event.data.locus)) {
-      if (!get(this, 'config.enableExperimentalGroupCallingSupport') && !isCall(event.data.locus)) {
-        return;
-      }
+    // We only want to handle calls we are not aware of
+    if (this.emittedCalls.get(makeInternalCallId(event.data.locus))) {
+      return;
+    }
 
-      const call = Call.make({
-        locus: event.data.locus
-      }, {
-        parent: this.spark
-      });
+    // Create call object and store in emittedCalls
+    const call = Call.make({
+      locus: event.data.locus
+    }, {
+      parent: this.spark
+    });
+    this.emittedCalls.add(call);
 
-      this.trigger('call:incoming', call);
+    // Trigger events as necessary
+    this.trigger(events.CALL_CREATED, call);
```

Comment:
Do I correctly read this as emit `CALL_CREATED` once for every call and further emit `CALL_INCOMING` once if appropriate?